requirements.txt

```diff
@@ -14,7 +14,7 @@ timm
 scikit-image
 gradio
 controlnet_aux==0.0.6
-tensorflow==2.8.0
+#tensorflow==2.8.0
```

Comment:
have we tested this in a clean environment?

Reply:
Yes, it has been tested  successfully.